pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2643,25 +2643,25 @@ func checkAndSetIoMember(ctx *domainContext, ib *types.IoBundle, isPort bool, pu
 	ib.IsPort = false
 	changed := false
 	if isPort {
-		log.Warnf("checkAndSetIoBundle(%d %s %s) part of zedrouter port\n",
-			ib.Type, ib.Name, ib.AssignmentGroup)
+		log.Warnf("checkAndSetIoMember(%d %s %s %s) part of zedrouter port\n",
+			ib.Type, ib.Name, ib.Ifname, ib.AssignmentGroup)
 		ib.IsPort = true
 		changed = true
 		if ib.UsedByUUID != nilUUID {
-			log.Errorf("checkAndSetIoMember(%d %s %s) used by %s",
-				ib.Type, ib.Name, ib.AssignmentGroup,
+			log.Errorf("checkAndSetIoMember(%d %s %s %s) used by %s",
+				ib.Type, ib.Name, ib.Ifname, ib.AssignmentGroup,
 				ib.UsedByUUID.String())
 
 		} else if ib.IsPCIBack {
-			log.Infof("checkAndSetIoMember(%d %s %s) take back from pciback\n",
-				ib.Type, ib.Name, ib.AssignmentGroup)
+			log.Infof("checkAndSetIoMember(%d %s %s %s) take back from pciback\n",
```

Comment:
How about updating all the log messages for this function to print 4 items inside the parenthesis and print them in the same order. No reason to have them be slightly different.

Reply:
Done..